File: src/test/unit/PersonTest.java
package test.unit;

import static org.junit.Assert.assertEquals;

import org.junit.Test;

import person.Student;

public class PersonTest {

	@Test
	public void test_person_first_name() {
		Student testPersonClass = new Student("John","Green",null,null,null,null);
		
		testPersonClass.setFirstName("John");
		assertEquals("John", testPersonClass.getFirstName());
	}
	@Test
	public void test_person_last_name() {
		Student testPersonClass = new Student("John","Green",null,null,null,null);
		
		testPersonClass.setLastName("Green");
		assertEquals("Green", testPersonClass.getLastName());
		
	}
	@Test
	public void test_person_full_name() {
		Student testPersonClass = new Student("John","Green",null,null,null,null);
		
		testPersonClass.setFirstName("John");
		
		testPersonClass.setLastName("Green");
		
		assertEquals("John Green", testPersonClass.getFullName());
	}
}

File: pythonCode/PersonTest.py
import unittest
from Person import Person
class PersonTest(unittest.TestCase):
    # generated source for class PersonTest 
    def test_person_first_name(self):
        # generated source for method test_person_first_name 
        testPerson = Person()
        testPerson.setFirstName("testFName")
        self.assertEqual("testFName", testPerson.getFirstName())
    def test_person_last_name(self):
        # generated source for method test_person_last_name 
        testPerson = Person()
        testPerson.setLastName("testLName")
        self.assertEqual("testLName", testPerson.getLastName)
    def test_person_full_name(self):
        # generated source for method test_person_full_name 
        testPerson = Person()
        testPerson.setFirstName("testFName")
        testPerson.setLastName("testLName")
        self.assertEqual("testFName testLName", testPerson.getFullName)
if __name__ == '__main__':
    unittest.main()